src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -693,6 +888,23 @@ public void updateUnsolvedClassWithFields(
     }
   }
 
+  /**
+   * This method checks if a variable is local.
+   *
+   * @param fieldName the name of the variable
+   * @return true if that variable is local
+   */
+  public boolean isALocalVar(String fieldName) {
```

Comment:
It is confusing that the name of the parameter to this method is `fieldName`, since the parameter might or might not be the name of a field. I suggest `variableName` or something like that, instead.